src/UNL/MediaHub/DefaultHomepage.php

```diff
@@ -22,12 +22,54 @@ function getCacheKey()
 
     function run()
     {
+        
+    }
+    
+    function getTopMedia()
+    {
         $options = array(
-            'limit'   => 6,
+            'limit'   => null,
             'orderby' => 'popular_play_count'
         );
-        $this->top_media = new UNL_MediaHub_MediaList($options);
-        $this->top_media->run();
+        $top_media = new UNL_MediaHub_MediaList($options);
```

Comment:
I'm not worried about this method being used more than once in the future.  It has a very specific purpose on a very specific class.